common/src/main/java/com/wynntils/core/telemetry/type/TelemetryGameVersion.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.telemetry.type;
+
+/**
+ * This enum represents the game version that the telemetry is being sent from.
+ * Not every version may have a telemetry version, and telemetry versions may be skipped.
+ * This enum is only updated whenever relevant changes are made to the game.
+ */
+public enum TelemetryGameVersion {
+    VERSION_203_HOTFIX_4("2.0.3 Hotfix 4");
```

Comment:
I think this is conceptually wrong, unfortunately. :-( You cannot deduce the version of Wynncraft based on the version of Wynntils. Someone can play an non-updated version of Wynntils on a more recent Wynncraft version.

In fact, as long as Wynncraft does not add a way to detect version in-game, I am afraid we'll need an Athena service (or on github) that returns the current Wynncraft game version, and that you needs to poll at startup.

Reply:
I don't think so. This enum shows that this Wynntils version was made for this version of Wynncraft. If you want to contribute data, update your mod. If we want to collect data, we can bump this version, if and when needed.

If someone plays on a version that has the correct enum, we can assume that the data they collect is correct. At least, if they don't try to fake the data..

Reply:
If I had any trust in Athena at this point..

If this was something we would immediately send to a server, yes, this sounds good. But this data is to be hand collected in discord at the moment.. Even is someone would code the athena part, it is really unstable to receive this much new info. 

I can see the concern though. Maybe keeping this until we don't have a proper server to receive this data..? Again, that could take a while, so I also don't want to leave this in a bad place either.

Reply:
Ah, I see. I did not understand that the data were to be hand collected. So this is basically a new take on my old "record NPC locations" mode in legacy.

Fair enough, let's do it like this for now then.